src/lib/QtWidgets/QTabWidget.ts

```diff
@@ -39,15 +39,46 @@ export class QTabWidget extends QWidget<QTabWidgetSignals> {
         }
         super(native);
     }
-
-    addTab(page: QWidget, icon: QIcon, label: string): number {
-        const index = this.native.addTab(page.native, icon.native, label);
+    /**
+     * Adds a tab to the TabBar of the widget.
+     * If you include an ampersand (`&`) in the label, the next character will become the shortcut to this tab.
+     * Eg: the label "Bro&wse" will assign ALT+W to focus on this tab.
+     * @param page The widget that will become the body of the tab.
+     * @param icon The icon of the tab (optional).
+     * @param label The label of the tab. 
+     * @returns The index of the tab.
+     */
+    addTab(page: QWidget, icon: QIcon | string | undefined, label: string | undefined): number {
```

Comment:
In these cases I would call this parameter `iconOrLabel` which is closer to the truth.

Reply:
Sure. Let me change it quickly.